integration_tests/test_checkov_platform_only_policies.py

```diff
@@ -0,0 +1,33 @@
+import os
+import platform
+import sys
+import unittest
+import re
+
+from checkov.common.models.consts import ckv_check_id_pattern
+
+current_dir = os.path.dirname(os.path.realpath(__file__))
+
+
+class TestCheckovPlatformOnlyPolicies(unittest.TestCase):
+
+    def test_no_ckv_ids_api_key(self):
+        checks_list_path = os.path.join(current_dir, '..', 'checkov_checks_list.txt')
+        if sys.version_info[1] == 7 and platform.system() == 'Linux':
+            with open(checks_list_path, encoding='utf-8') as f:
+                for i, line in enumerate(f):
+                    if i in [0, 1]:
+                        # skip the header lines
+                        continue
+                    line = "".join(line.split())
+                    if type(line) == str and line:
```

Comment:
```suggestion
                    if line and isinstance(line):
```
most of the time `isinstance` usage is preferred compared to `type` for checking the of a variable.

Reply:
thanks for the tip!